### PR TITLE
Fixes missiles being able to ricochet

### LINKED ISSUE #29261: Shuttle walls reflect missiles
I don't think that beam and bullet reflection should work on missiles

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
fix: Missiles can no longer ricochet off of shuttle walls, etc.
/:cl:

Fixes #29261.
They're _missiles._

### CODE REVIEW

File: code/modules/projectiles/projectile/special.dm [hunk 1]
```diff
@@ -69,6 +69,7 @@
 	desc = "Boom"
 	icon_state = "missile"
 	damage = 30
+	ricochets_max = 0 //it's a MISSILE
```
Comment (Cyberboss) [inline code/modules/projectiles/projectile/special.dm]: There's a different var that @Fox-McCloud added for the ability for something to ricochet

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/projectiles/projectile/special.dm (hunk 1 above)
Comment (Xhuis) [inline code/modules/projectiles/projectile/special.dm]: I'm not able to see it, but I might just be dumb.
https://github.com/tgstation/tgstation/blob/master/code/modules/projectiles/projectile.dm#L140

Comment (Cyberboss) [inline code/modules/projectiles/projectile/special.dm]: https://github.com/tgstation/tgstation/blob/master/code/modules/projectiles/projectile.dm#L43

Comment (Xhuis) [inline code/modules/projectiles/projectile/special.dm]: I'm pretty sure that's for reflector vests, as the ricochet code doesn't check it at all.
Still, I'll take a look.

Comment (Cyberboss) [inline code/modules/projectiles/projectile/special.dm]: #29081 for reference

**Review by Cyberboss — APPROVED** [review]

**Comment by Repukan** [comment]
How about bullets

File: code/modules/projectiles/projectile/special.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/projectiles/projectile/special.dm]: @Cyberboss 

That's for when something that returns `isReflect`, not for wall ricochets.

**Comment by Fox-McCloud** [comment]
>How about bullets

. . . ok.

**Comment by DoctoreMad** [comment]
What about bullets?
If they stop ricocheting then that'll be a incredible loss.

**Comment by Xhuis** [comment]
I didn't affect bullets in any way. They still ricochet.

**Comment by kevinz000** [comment]
why the fuck not :^)

**Pull request merged**